components/form-token-field/index.js

```diff
@@ -469,6 +481,9 @@ class FormTokenField extends Component {
 
 		return (
 			<div { ...tokenFieldProps } >
+				<label htmlFor={ `components-form-token-input-${ instanceId }` } className="components-form-token__howto">
```

Comment:
Should we be rendering this if there's an empty `placeholder` ? Maybe want a more useful default `placeholder` ?